keps/sig-node/20190212-ephemeral-containers.md

```diff
@@ -129,10 +130,15 @@ This confers several advantages:
 1.  **Smaller image size** reduces resource usage and speeds deployments.
 
 The disadvantage of using containers built `FROM scratch` is the lack of system
-binaries provided by an Operating System image makes it difficult to
+binaries provided by a Linux distro image makes it difficult to
 troubleshoot running containers. Kubernetes should enable one to troubleshoot
 pods regardless of the contents of the container images.
 
+On Windows, the minimal [Nano Server](https://hub.docker.com/_/microsoft-windows-nanoserver)
+image is the smallest available, which still retains the `cmd` shell and some
+common tools such as `curl.exe` which could be used to retrieve other binaries or
+scripts as needed. Windows cannot build containers `FROM scratch`.
```

Comment:
Since even the minimal Windows has a shell and `curl` installed, should we suggest users to simply download and install debugging tools as an alternative?